app/models/post.rb

```diff
@@ -70,14 +111,26 @@ def self.for_gratitude_and_praise
       .where('post_types.name ILIKE ? OR post_types.name ILIKE ?', '%gratitude%', '%praise%')
   end
 
-  def self.bookmarked
-    where(bookmarked: true)
-  end
-
   def self.in_last_calendar_year
     where('date BETWEEN ? AND ?', Time.zone.today - 365, Time.zone.today)
   end
 
+  def self.to_csv
```

Comment:
This is where all of the magic happens here. The weirdness with the attributes is because I wanted to include more data than just what's on the `post` object itself. Line 126 is a big chain of shame. I did not do the work to figure out a better way.